ui/app/AppLayouts/Communities/controls/FeeRow.qml

```diff
@@ -0,0 +1,109 @@
+import QtQuick 2.15
```

Comment:
I'm getting some warnings on this file. Might be different on other Qt versions. Do you also see them in storybook?
```
Qt Quick Layouts: Detected recursive rearrange. Aborting after two iterations.
```

Reply:
solved!